docker-compose.yaml

```diff
@@ -1,19 +1,37 @@
 version: '3'
+volumes:
+    elections:
 services:
+  ldap:
+    image: jenkinsciinfra/mock-ldap:latest
+    ports:
+        - 636:636
+        - 389:389
+
   sandbox:
     build: .
     image: accountapp:latest
+    volumes:
+      - elections:/var/log/accountapp/elections
     depends_on:
       - ldap
     env_file: env.sandbox
+    environment:
+        - ELECTION_CLOSE=2038/11/23
```

Comment:
I think we should make sure to use more consistent/standard date formats, like ISO-8601, e.g. : `2030-11-23`

I recognize that this is a bit late on this feedback, but I'm hoping it's relatively easy to fix up

Reply:
I guess, the goal was to be consistent with the rest of the application.
But let changes that